### PR TITLE
remove mentions of IRC, replace with slack

### PR DESCRIPTION
**Pull request by irabinovitch**
remove mentions of IRC, replace with slack

### CODE REVIEW

**Comment by irabinovitch** [comment]
@jhotta  please do the same on the JP docs.

File: layouts/_contact_info.html [hunk 1]
```diff
@@ -11,10 +11,8 @@ <h4 id="support">Web Support</h4>
 <h4 id="email">By Email</h4>
 You can also contact our Support Team via email at <a href="mailto:support@datadoghq.com">support@datadoghq.com</a>.
 
-<h4 id="irc">Over IRC</h4>
-Reach out to our team and other Datadog users in our IRC channel, <a href="irc://irc.freenode.net/datadog">#datadog on freenode</a>.
-(There's a <a href="http://webchat.freenode.net/?randomnick=1&channels=datadog&prompt=1"> web chat client, too</a>.)
-
+<h4 id="slack">Over Slack</h4>
+Reach out to our team and other Datadog users on <a href="http://slack.datadoghq.com">Slack</a>.
```
Comment (jyee) [inline layouts/_contact_info.html]: Shouldn't the URL be http://chat.datadoghq.com?

Comment (irabinovitch) [inline layouts/_contact_info.html]: yep

**Comment by irabinovitch** [comment]
@jyee look ok now?

**Pull request merged**